src/content/docs/ecosystem/index.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Ecosystem
+description: The SKALE Ecosystem is comprised of builders, delegators, and validators.
+sidebar:
+    order: 2
+---
+
+import { LinkCard, CardGrid } from '@astrojs/starlight/components';
+
+## The SKALE Ecosystem
+<CardGrid>
+    <LinkCard
+        title="Builders"
+        href="/builders"
```

Comment:
Was this builder card placed here on purpose? I would remove it but let me know your thoughts

Reply:
Yes -- placed on purpose as builders are part of the ecosystem.